slides/content/02-kotlin/_index.md

```diff
@@ -106,21 +106,18 @@ foo(b = "bar") // OK, same as foo(0, "bar")
 # Kotlin 101
 ## Top level functions
 
-Kotlin supports top level functions
+Kotlin supports top level functions as well as Scala (3)
```

Comment:
Ok in Scala 3.0 too
https://scastie.scala-lang.org/cric96/zvJbAXGyT1KBGSgsnjAbLw